{
    "id": "CVE-2021-4118",
    "product": "pytorchlightning/pytorch-lightning",
    "version": "< 1.6 ",
    "vulnerability": [
        "CWE-502 Deserialization of Untrusted Data"
    ],
    "description": "pytorch-lightning is vulnerable to Deserialization of Untrusted Data",
    "poc": {
        "reference": [
            "https://huntr.dev/bounties/31832f0c-e5bb-4552-a12c-542f81f111e6"
        ],
        "github": [
            "No PoCs found on GitHub currently."
        ]
    }
}